### PR TITLE
PXB-2872 Document quicklz compression deprecation (8.0)

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
modified:   docs/backup_scenarios/compressed_backup.md
	modified:   docs/howtos/recipes_xbk_compressed.md
	modified:   docs/installation/apt_repo.md
	modified:   docs/installation/yum_repo.md
	modified:   docs/xbstream/xbstream.md
	modified:   docs/xtrabackup_bin/backup.streaming.md
	modified:   docs/xtrabackup_bin/xbk_option_reference.md

### CODE REVIEW

File: docs/backup_scenarios/compressed_backup.md [hunk 1]
```diff
@@ -5,10 +5,14 @@ backup can be compressed or decompressed with *xbstream*.
 
 ## Create compressed backups
 
+!!! note
+
+    Starting with Percona XtraBackup 8.0.31-24 using qpress/quicklz to compress backups is deprecated and may be removed in future versions. We recommend using either `lz4` or Zstandard (`ZSTD`) compression algorithms.
```
Comment (patrickbirch) [inline docs/backup_scenarios/compressed_backup.md]: LZ4

File: docs/installation/apt_repo.md [hunk 2]
```diff
@@ -48,6 +48,10 @@ with the *percona-release* package configuration tool.
 
 6. To make compressed backups, install the `qpress` package:
```
Comment (patrickbirch) [inline docs/installation/apt_repo.md]: This step should be updated to remove the qpress option.

Comment (alina-derkach-oaza) [reply docs/installation/apt_repo.md]: Updated

File: docs/backup_scenarios/compressed_backup.md (hunk 1 above)
Comment (alina-derkach-oaza) [reply docs/backup_scenarios/compressed_backup.md]: Updated

**Review by patrickbirch — APPROVED** [review]
OK

**Pull request merged**